cmd/zpool/zpool_main.c

```diff
@@ -1766,7 +1766,7 @@ print_import_config(const char *name, nvlist_t *nv, int namewidth, int depth,
  */
 static void
 print_logs(status_cbdata_t *cb, zpool_handle_t *zhp, nvlist_t *nv,
-    int namewidth, boolean_t verbose, int name_flags)
+    int namewidth, int name_flags)
```

Comment:
Let's avoid any crazy casting tricks, but I do like the notion of putting more of this state in the `status_cbdata_t`.  How about instead of adding a new `import_cbdata_t` which is largely redundant with the `status_cbdata_t` we just extend it as needed to cover both cases.